iw/filter_gui.py

```diff
@@ -878,7 +890,7 @@ def _active(info):
                                  list(rs.get_members()))
                 members = map(lambda m: m.get_devpath(), members)
                 for d in raids:
-                    if udev_device_get_name(d) in members:
+                    if udev_device_get_name(d) in members and d not in mdraids:
                         fstype = udev_device_get_format(d)
                         sysfs_path = udev_device_get_sysfs_path(d)
                         break
```

Comment:
I think I want to replace this hunk with the following:

```diff
@@ -878,6 +890,9 @@ class FilterWindow(InstallWindow):
                                  list(rs.get_members()))
                 members = map(lambda m: m.get_devpath(), members)
                 for d in raids:
+                    if d in mdraids:
+                        break
+
                     if udev_device_get_name(d) in members:
                         fstype = udev_device_get_format(d)
                         sysfs_path = udev_device_get_sysfs_path(d)

```

I will retest with the change and update the PR when it checks out.